pkg/flow/ent/schema/logtag.go

```diff
@@ -0,0 +1,28 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"github.com/google/uuid"
+)
+
+// LogMsg holds the schema definition for the logmsg entity.
+type LogTag struct {
+	ent.Schema
+}
+
+// Fields of the LogMsg.
+func (LogTag) Fields() []ent.Field {
+	return []ent.Field{
+		field.UUID("id", uuid.UUID{}).Default(uuid.New).Immutable().StorageKey("oid").StructTag(`json:"-"`),
+		field.String("type"),
```

Comment:
Should we create a unique index on the compound logmsg-type? I'm thinking duplicates here should be impossible, right?

Reply:
A unique index doesn't have to replace the ID. But I think the answer to my original question isn't clear. We need to figure out how we're going to handle log inheritance first. It's plausible that `type` could have duplicates.

Reply:
log inheritance can be solved by storing the "call-path" in each instance, this is what i need the `parents` array in the instance-schema for. This "call-path" contains the IDs of all parent instances. For example: 
```
instance a -> children: instance b, instance c
instance b -> children: instance d, instance e
```
The  `parents` array / Call-Path for instance e is: `a,b,d,e` 
Using this information we can add a foreign key  / Edge to all parent instances when we insert a new logmsg into the database. 

So that when we want all logs for instance b and all its child's we can do a select query for all logmsg with a  foreign key  / Edge to instance b.

Reply:
And regarding the logtag logmsg compound type: I see logtag as attributes/ properties to describe instances/ database objects so they a not really a compound to logmsgs.  For example if we need to display all logs which where printed at state `XY`, from a given instanceID and workflow `A`, we can search in the database for all logmsgs connected to this instance which have logtags with the type workflow value `A` and logtags with type state value `xy`